# Data FROM MAVLink stream TO companion computer

uint64 timestamp	# Local time since system start (microseconds)
bool output_on		# Acknowledge on
bool output_off		# Acknowlege off
